Resources/Audio/Voice/Cluwne/attributions.yml

```diff
@@ -0,0 +1,12 @@
+- files: ["sound_voice_cluwnelaugh1.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "cluwne laugh 1 uploaded by ZeWaka and taken from goonstation github"
+  source: "https://github.com/goonstation/goonstation/blob/master/sound/voice/cluwnelaugh1.ogg"
```

Comment:
the link here should be a commit of the file at the time you took it, so something like this 
https://github.com/goonstation/goonstation/blob/4059e4be90832b02b1228b1bee3db342094e4f1e/sound/voice/cluwnelaugh1.ogg
Also this is from goonstation, which means you are gonna have to ping ZeWaka here and ask them to corroborate the license and copyright (same to the rest of the file)

Reply:
should be resolved since the sfx now fro freesound.org